scripts/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+BASEDIR=$(pwd)
+set -e
+make clean install
+sudo rm -rf ./build/networks
```

Comment:
we really should need to sudo this?

Reply:
Regretfully so, at least on my machine, rm doesn't want to touch the crypto stuff

Reply:
sudo is not required on my machine

Reply:
@Joe-mcgee can you give me an output of what the permissions get set to on this folder and its children? We might need to update the sifgen tool and set the permissions

Reply:
@utx0  sure thing 
![sofnoded-permissions](https://user-images.githubusercontent.com/24814085/99283714-9cbaf880-27f2-11eb-9dc1-e5dcc64ba085.png)